flag_engine/utils/json_encoder.py

```diff
@@ -0,0 +1,16 @@
+import json
+import decimal
+
+
+class DecimalEncoder(json.JSONEncoder):
```

Comment:
Feels like _maybe_ this should live in a more specific module location: `flag_engine.utils.json.encoders.DecimalEncoder`. 

Perhaps this is overkill but I just don't want to force ourselves into a corner in case we need more utils in the future.